NTPClient.h

```diff
@@ -30,6 +31,7 @@ class NTPClient {
     NTPClient(UDP& udp);
     NTPClient(UDP& udp, int timeOffset);
     NTPClient(UDP& udp, const char* poolServerName);
+    NTPClient(UDP& udp, IPAddress poolServerIP);
     NTPClient(UDP& udp, const char* poolServerName, int timeOffset);
     NTPClient(UDP& udp, const char* poolServerName, int timeOffset, int updateInterval);
```

Comment:
@sheffieldnick thanks for submitting this pull request!

The change above looks good to me. However, what do you think about adding the IPAddress option to the other constructors that accept a string?

```c
    NTPClient(UDP& udp, IPAddress poolServerIP, int timeOffset);
    NTPClient(UDP& udp, IPAddress poolServerIP, int timeOffset, int updateInterval);
```